pages/contact.html

```diff
@@ -41,5 +41,18 @@
     </a>
 </div>
 </div>
+
+<div class="ui main text container">
+    <h1 class="ui header">School Contact Details</h1>
+    <p>Email School: info@mountsioncbssecondary.ie</p>
+    <p>Telephone Number: 051 377378</p>
+    <p>Address:</p>
+    <p>Mount Sion CBS Secondary School,</p>
+    <p>Barrack Street,</p>
+    <p>Waterford City.</p>
+    <h2 class="ui header">School Location</h2>
+    <iframe src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d9769.151915664062!2d-7.1166274!3d52.2563154!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0xf8355ef4efe9f1a4!2sMount+Sion+CBS+Secondary+School!5e0!3m2!1sen!2sie!4v1523822538739" width="600" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>
```

Comment:
I would suggest maybe changing `width` to be equal to `100%` it will fill it out to the size of the container no matter the size of the screen.